app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
It's not a good practice to modify the class state inside the constructor (`__init__`). The constructor should be used only for initializing the object. This is a code quality issue.